### PR TITLE
add project to plannerPlanContainer

### PR DESCRIPTION
**Pull request by victorliu-microsoft**
This change adds project to Planner's PlanContainer resource type.

### CODE REVIEW

File: api-reference/beta/resources/plannerplancontainer.md [hunk 1]
```diff
@@ -20,7 +20,8 @@ Planner currently supports the container types listed in the following table. Wh
 |Type|Description|Path to the resource|
 |----|-----------|--------------------|
 |group| Plan is contained by a [group](group.md).| https://graph.microsoft.com/beta/groups/&lt;id&gt;|
-|roster| Plan is contained by a [plannerRoster](plannerroster.md) | https://graph.microsoft.com/beta/planner/rosters/&lt;id&gt;|
+|roster| Plan is contained by a [plannerRoster](plannerroster.md). | https://graph.microsoft.com/beta/planner/rosters/&lt;id&gt;|
+|project| Plan is contained by a Project from [Microsoft Project](/project). | Microsoft Project currently doesn't have a resource path on Graph, Project details and members can be managed from https://project.microsoft.com. |
```
Comment (TarkanSevilmis) [inline api-reference/beta/resources/plannerplancontainer.md]: Microsoft Graph

Comment (TarkanSevilmis) [inline api-reference/beta/resources/plannerplancontainer.md]: Should be a full stop here

Comment (TarkanSevilmis) [inline api-reference/beta/resources/plannerplancontainer.md]: Microsoft Project for the web?

Comment (TarkanSevilmis) [inline api-reference/beta/resources/plannerplancontainer.md]: Need absolute url here

**Review by TarkanSevilmis — APPROVED** [review]

**Comment by victorliu-microsoft** [comment]
Schema review change log: https://dev.azure.com/msazure/One/_git/AD-AggregatorService-Workloads/pullrequest/6665314

File: api-reference/beta/resources/plannerplancontainer.md [hunk 2]
```diff
@@ -20,7 +20,8 @@ Planner currently supports the container types listed in the following table. Wh
 |Type|Description|Path to the resource|
 |----|-----------|--------------------|
 |group| Plan is contained by a [group](group.md).| https://graph.microsoft.com/beta/groups/&lt;id&gt;|
-|roster| Plan is contained by a [plannerRoster](plannerroster.md) | https://graph.microsoft.com/beta/planner/rosters/&lt;id&gt;|
+|roster| Plan is contained by a [plannerRoster](plannerroster.md). | https://graph.microsoft.com/beta/planner/rosters/&lt;id&gt;|
+|project| Plan is contained by a Project from [Microsoft Project for the web](https://learn.microsoft.com/project-for-the-web/projectforweb-admin-home). | Microsoft Project currently doesn't have a resource path on Microsoft Graph. Project details and members can be managed from https://project.microsoft.com. |
```
Comment (isvargasmsft) [inline api-reference/beta/resources/plannerplancontainer.md:24]: Please update line 30 with the new member.

I am taking the liberty of adding the new member to the enum definition.

**Review by isvargasmsft — APPROVED** [review]

**Pull request merged**